include/swift/AST/DiagnosticsSema.def

```diff
@@ -4052,6 +4054,8 @@ NOTE(because_rethrows_argument_throws,none,
 NOTE(because_rethrows_default_argument_throws,none,
      "call is to 'rethrows' function, but a defaulted argument function"
      " can throw", ())
+NOTE(because_rethrows_default_conformance_throws,none,
+     "call is to 'rethrows' function, but a conformance can throw", ())
```

Comment:
"a conformance has a throwing witness" perhaps?

Reply:
that will be the first case of witness being used in diagnostics, are we certain that is grok-able enough?
perhaps I can write it as "a conformance has a throwing method satisfying a protocol"?